lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -21,8 +21,10 @@
 from .displayCustomControl import DisplayCustomControl
 from .materialCustomControl import MaterialCustomControl
 from .metadataCustomControl import MetadataCustomControl
+from .lightCustomControl import LightLinkingCustomControl
```

Comment:
I think you could wrap this in a try/except block and set a boolean variable such as `lightLinkingSupported`. Then at line 649 below only do it when that variable is True.